pkg/cluster/cluster.go

```diff
@@ -188,6 +190,14 @@ func (c *Cluster) checkClientAuthUpdate() error {
 			c.cluster.SetClientAuthSecretResourceVersion(result.ResourceVersion)
 			return c.updateConfigSecret()
 		}
+	} else if c.cluster.Spec.Auth.ClientsAuthFile != "" {
+		hasher := sha1.New()
```

Comment:
So it seems like, from the conversation in https://github.com/nats-io/nats-operator/issues/160, that these changes are unnecessary and if the file path is changed, the config update will happen due to `updateCluser`?

Reply:
that's right, if the file path changes via the CRD update then that would trigger the reload as well